test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,23 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(500);
```

Comment:
It seems like 100ms was causing the intermittent failure in CI in the first place, unless that can be handled in platformTimeout.
cc @mhdawson